profiles/common/modules/features/ec_embedded_video/includes/ec_embedded_video.formatters.inc

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @file
+ * File formatters for Platform needs.
+ */
+
+/**
+ * Implements hook_file_formatter_info_alter().
+ *
+ * It alters file_formatter_info to add the "Wrapper free formatters for the
+ * different Video-sharing services (YouTube, Vimeo, Dailymotion and
+ * AV portal).
+ */
+function ec_embedded_video_file_formatter_info_alter(&$info) {
```

Comment:
@drupol, I tested what you suggest and as I feared, I got an infinitive loop.

I thought wrongly that the info alter was cached but it is only stored in the cache static... like "module_invoke_all". So, I miss maybe something but I have the impression that using alter or not have the same effect here.

If you want we can discuss about that during the day.